<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>TeX Gyre Adventor</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Automata Finito Determinado</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableViewTransiciones">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLlenartransiciones">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Llenar Trans</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitleTransiciones">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>TABLA DE TRANSICIONES</string>
    </property>
   </widget>
   <widget class="Line" name="lineHorizontal1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelINPUT">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>INPUT</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMAQUINA">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>MAQUINA</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstado">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlfabeto">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alfabeto</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstadoInicial">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado I</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Estado">
     <item>
      <widget class="QLineEdit" name="txtEstado"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddEstado">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Alfabeto">
     <item>
      <widget class="QLineEdit" name="txtAlfabeto"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddAlfabeto">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelEstadoAceptacion">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado A</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="EstadoInicial">
     <item>
      <widget class="QLineEdit" name="txtEstadoInicial"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddEstadoInicial">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="EstadoAceptacion">
     <item>
      <widget class="QLineEdit" name="txtEstadoAceptacion">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddEstadoAceptacion">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="txtEstados">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstados">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Estados</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlfabetos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Alfabeto</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstadoInicial2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado I</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstadosAceptaciones">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado A</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCadena">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Cadena</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlfabetos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstadoInicial2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstadosAceptaciones">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="lineHorizantal2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineHorizantal3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCadena">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVerificar">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Verificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFormarTransiciones">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>240</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Formar Trans</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>82</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="ok1">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok2">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok3">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok4">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionDot"/>
    <addaction name="actionIniciar"/>
    <addaction name="action_Clear"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDot"/>
   <addaction name="actionIniciar"/>
   <addaction name="action_Clear"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/close.jpeg</normaloff>:/rec/close.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionDot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/dot.png</normaloff>:/rec/dot.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Dot</string>
   </property>
  </action>
  <action name="actionIniciar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/Im치genes/images_qt_menu_toolbar/iniciar.jpeg</normaloff>:/rec/Im치genes/images_qt_menu_toolbar/iniciar.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Iniciar</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/Im치genes/images_qt_menu_toolbar/clear.jpeg</normaloff>:/rec/Im치genes/images_qt_menu_toolbar/clear.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
